api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/ClaimantCategoryCalculator.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.dhsc.htbhf.claimant.reporting;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PregnancyEntitlementCalculator;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.util.List;
+
+import static uk.gov.dhsc.htbhf.claimant.reporting.payload.ClaimantCategory.NOT_PREGNANT_WITH_CHILDREN;
+import static uk.gov.dhsc.htbhf.claimant.reporting.payload.ClaimantCategory.PREGNANT_AND_UNDER_16;
+import static uk.gov.dhsc.htbhf.claimant.reporting.payload.ClaimantCategory.PREGNANT_WITH_CHILDREN;
+import static uk.gov.dhsc.htbhf.claimant.reporting.payload.ClaimantCategory.PREGNANT_WITH_NO_CHILDREN;
+
+@Component
+@RequiredArgsConstructor
+public class ClaimantCategoryCalculator {
+
+    private final PregnancyEntitlementCalculator pregnancyEntitlementCalculator;
+
+    /**
+     * Determines the {@link uk.gov.dhsc.htbhf.claimant.reporting.payload.ClaimantCategory} for a given claimant and their children's date of birth.
+     *
+     * @param claimant               the claimant to determine the category of
+     * @param datesOfBirthOfChildren the dates of birth of the claimant's children
+     * @param timestamp the timestamp to use when checking a claimant/child's age
+     * @return claimant's determined category
+     */
+    public String determineClaimantCategory(Claimant claimant, List<LocalDate> datesOfBirthOfChildren, LocalDateTime timestamp) {
```

Comment:
I suggest this method returns a ClaimantCategory, and always returns a value. The logic can be simplified a bit:
```
if (isClaimantPregnant(claimant)) {
	if (claimantAgeInYears < 16) {
		return PREGNANT_AND_UNDER_16;
	}
	if (hasChildren(datesOfBirthOfChildren)) {
	   return PREGNANT_WITH_CHILDREN;
	}
	return PREGNANT_WITH_NO_CHILDREN;
}
if (hasChildren(datesOfBirthOfChildren)) {
   return NOT_PREGNANT_WITH_CHILDREN;
}
// this could happen once a claimant has expired
return NOT_PREGNANT_NO_CHILDREN;
```

Reply:
Actually, we're also missing this category: Pregnant under 18/20FTE
I know we can't determine the under 20 FTE part, but we can determine under 18